doc/source/io.rst

```diff
@@ -4005,14 +4005,11 @@ The key functions are:
 .. currentmodule:: pandas.io.gbq
 
 .. autosummary::
-    :toctree: generated/
+   :toctree: generated/
 
-    read_gbq
-    to_gbq
-    generate_bq_schema
-    create_table
-    delete_table
-    table_exists
+   read_gbq
```

Comment:
it's OK (they just have to be indented consistenly, but does not matter how many spaces)